cards/document-standard/template.hbs

```diff
@@ -90,4 +93,20 @@
   </a>
 </div>
 {{/if}}
+{{/inline}}
+
+{{#*inline 'feedback'}}
```

Comment:
What do you think about making a feedbackWrapper partial? It looks like every cards follows this same structure, but with different class names. We may be able to make a partial which allows the card class name to be passed in as a param, and then we wouldn't have to define this inline helper in every card.